cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
```

Comment:
Are commands the recommended way to add functionality? Why would I use a command over just writing a function with the same bodies as below? They'd be more succinct and less magical.

Reply:
I saw the guide doing it in here so I think its probably a good practice to add it in that file.

Reply:
I believe it is to enable TS "support" for a particular file. In TS, we don't need this.